src/scripts/main.js

```diff
@@ -1 +1,14 @@
 'use strict';
+
+const largeImage = document.getElementById('largeImg');
+const galleryList = document.querySelector('.gallery__list');
+
+galleryList.addEventListener('click', (currentEvent) => {
+  const link = currentEvent.target.closest('.list-item__link');
```

Comment:
Consider checking if 'currentEvent.target' is not null before calling 'closest' to avoid potential errors if the event target is not an element.